numerics/polynomial_evaluators_body.hpp

```diff
@@ -0,0 +1,229 @@
+#pragma once
+
+#include "numerics/polynomial_evaluators.hpp"
+
+#include <tuple>
+
+namespace principia {
+namespace numerics {
+namespace internal_polynomial_evaluators {
+
+namespace {
+
+// Greatest power of 2 less than or equal to n.  8 -> 8, 7 -> 4.
+constexpr int FloorPowerOf2(int const n) {
```

Comment:
RoundToward0To1Bit

Reply:
Or FloorToPowerOf2, but there needs to be a To somewhere

Reply:
Done.